jep/1/README.adoc

```diff
@@ -0,0 +1,754 @@
+
+Once the champion has asked the Jenkins community as to whether an idea has any
+chance of acceptance, a draft JEP should be presented to jenkinsci-dev@. This
+gives the author a chance to flesh out the draft JEP to make sure it is
+properly formatted, of high quality, and to address initial concerns about the
+proposal.
+
+[[submission]]
+==== Creating a JEP Submission
+
+Following a discussion on jenkinsci-dev@,
+the proposal should be turned into as a JEP submission and submitted
+via a GitHub pull request to this repository footnoteref:[repo].
+The submission must be written in JEP style as described below,
+otherwise, it will fail review immediately
+(although minor errors may be corrected by the editors).
+
+To submit a JEP for approval as Draft, the JEP author should:
+
+. Fork the JEP repository footnoteref:[repo].
+. Create a new branch called `jep-submission` their fork
+  (`git checkout -b jep-submission`).
+. Copy the folder `jep-template/0000` to `jep/0000`.
+. Modify the template JEP in `jep/0000`
+  per the instructions in this JEP (which are also outlined in the template).
+. Commit and push the changes to their fork
+  and submit a pull request targeting the `jenkinsci/master` branch.
+
+A JEP editor will check the submission for conformance with
+JEP structure and formatting guidelines.
+Editors may make minor changes to make the submission meets
+the requirements for approval as a Draft JEP.
+If a JEP requires major changes, editors will add specific feedback
+and send the submission back to the author for revision.
+
+IMPORTANT: All submissions must go through pull request,
+even those by editors or contributors with "git push" privileges
+for the JEP repository footnoteref:[repo].
+
+The JEP editors will not unreasonably deny a JEP.
+Reasons for denying JEP "Draft" status include:
+
+* duplication of effort
+* being technically unsound
+* not providing enough information in the
+  "Motivation" or "Backwards Compatibility" sections
+* not in keeping with the Jenkins philosophy.
+
+The BDFL may be consulted during the approval phase,
+and is the final arbiter of a submission's approvability as a Draft JEP.
+
+[[approval]]
+==== Approval as Draft JEP
+
+Once JEP meets requirements for structure and formatting,
+the editors will approve the submission as a draft JEP
+by following the steps outlined in the
+<<approve-as-draft, editors' "Approve as Draft" section>>.
+When they are done, the Draft JEP will have an official JEP number and
+the sumbmission PR will have been merged to a matching folder and feature branch
+(i.e the
+`link:https://github.com/jenkinsci/jep/tree/jep-1/jep/1[jep/1]` folder in the
+`https://github.com/jenkinsci/jep/tree/jep-1/[jep-1]` branch).
+
+IMPORTANT: "Approval as Draft" is *not* the same as accepting the JEP.
+
+Editors are not the only ones who can approve as submission.
+Non-editor contributors who have "git push" privileges for the
+JEP repository footnoteref:[repo] may also approve submissions.
```

Comment:
It means jenkinsci GitHub organization admins, I'd guess.

Reply:
Doesn't have to be.   It means exactly what is says. 😄 
RESOLVED